otx/cli/builder/builder.py

```diff
@@ -140,27 +142,28 @@ def build_task_config(
         task_type: str,
         model_type: str = None,
         train_type: str = "incremental",
-        workspace_path: str = None,
-        otx_root=".",
+        workspace_path: Union[Path, str] = None,
+        otx_root: Union[Path, str] = ".",
     ):
         """Create OTX workspace with Template configs from task type.
 
         This function provides a user-friendly OTX workspace and provides more intuitive
         and create customizable templates to help users use all the features of OTX.
         task_type: The type of task want to get (str)
         model_type: Specifies the template of a model (str)
-        workspace_path: This is the folder path of the workspace want to create (os.path)
+        workspace_path: This is the folder path of the workspace want to create (Union[Path, str])
         """
 
         # Create OTX-workspace
         if workspace_path is None:
             workspace_path = f"./otx-workspace-{task_type}"
             if model_type:
                 workspace_path += f"-{model_type}"
-        os.makedirs(workspace_path, exist_ok=False)
+        workspace_path = workspace_path if isinstance(workspace_path, Path) else Path(workspace_path)
+        Path.mkdir(workspace_path, exist_ok=False)
```

Comment:
You could even do this, which is even easier:
```suggestion
        workspace_path.mkdir(exist_ok=False)
```

Reply:
I modified the code as suggested for a better quality!